test/metabase/query_processor_test/aggregation_test.clj

```diff
@@ -146,132 +141,6 @@
                (mt/run-mbql-query venues
                  {:aggregation [[:count] [:count]]})))))))
 
-
-;;; ------------------------------------------------- CUMULATIVE SUM -------------------------------------------------
-
-(deftest ^:parallel cumulative-sum-test
```

Comment:
Cumulative aggregations tests were moved to `query-processor-test.cumulative-aggregation-test`, and I added a lot more.